Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/mtl_command_buffer.mm

```diff
@@ -1725,12 +1725,20 @@ inline void PopDebugGroupCmd(id<MTLRenderCommandEncoder> encoder, IntermediateCo
     return *this;
 }
 
-RenderCommandEncoder &RenderCommandEncoder::setScissorRect(const MTLScissorRect &rect)
+RenderCommandEncoder &RenderCommandEncoder::setScissorRect(const MTLScissorRect &rect,
+                                                           id<MTLRasterizationRateMap> map)
 {
+    auto maxScissorRect =
+        MTLCoordinate2DMake(mRenderPassMaxScissorRect.width, mRenderPassMaxScissorRect.height);
+    if (map)
+    {
+        maxScissorRect = [map mapPhysicalToScreenCoordinates:maxScissorRect forLayer:0];
```

Comment:
Since forLayer is hardcoded to 0 here, does this mean this extension currently only works if the bound framebuffer only has one color attachment?

Reply:
This is one area that is TBD and still in development.